mipengine/node/node_logger.py

```diff
@@ -0,0 +1,25 @@
+import time
+from functools import wraps
+
+from celery.utils.log import get_task_logger
+
+
+def getLogger(name):
+    return get_task_logger(name)
+
+
+def log_method_call(func):
```

Comment:
Since we added back the celery logs, I think we should remove this wrapper from all the shared tasks.

We can leave this here, for future use in other methods.